### PR TITLE
Clarify the scope and intent of the SMS one-time code proposal. Addresses #18.

### PR DESCRIPTION
**Pull request by hober**

### CODE REVIEW

File: sms-one-time-code-format/README.md [hunk 1]
```diff
@@ -77,6 +89,16 @@ origin on which this code may be used.
 
 [schemelessly same site]: https://html.spec.whatwg.org/multipage/origin.html#schemelessly-same-site
 
+Adoption of this format would improve the reliability of systems which
+today heuristically extract one-time codes from SMS, with clear end-user
+benefit. It improves reliability of both extracting the code and also
+associating that code with an origin.
+
+Adoption of this proposal could improve the number of services on which
+a browser can offer SMS one-time code autofill, and could reduce the
```
Comment (rmondello) [inline sms-one-time-code-format/README.md]: I would say, “improve the number of services on which a browser can offer assistance with providing SMS one-time codes to websites (e.g. an AutoFill feature)”.

File: sms-one-time-code-format/README.md [hunk 2]
```diff
@@ -15,10 +15,13 @@
...
 
 - [Introduction](#introduction)
-- [Deficiencies of the status quo](#deficiencies-of-the-status-quo)
-- [Goal](#goal)
-- [Non-goal](#non-goal)
+  - [Deficiencies of the status quo](#deficiencies-of-the-status-quo)
+  - [Goals](#goals)
+  - [Non-goals](#non-goals)
 - [Proposal](#proposal)
+  - [](#)
```
Comment (rmondello) [inline sms-one-time-code-format/README.md]: What’s this line about?

File: sms-one-time-code-format/README.md (hunk 1 above)
Comment (hober) [reply sms-one-time-code-format/README.md]: Addressed in 4a932ca.

File: sms-one-time-code-format/README.md (hunk 2 above)
Comment (hober) [reply sms-one-time-code-format/README.md]: Noise. Fixed by running `doctoc` again in 4a932ca.

**Pull request merged**